desc/equilibrium/equilibrium.py

```diff
@@ -1311,6 +1468,45 @@ def Zb_lmn(self):
     def Zb_lmn(self, Zb_lmn):
         self.surface.Z_lmn = Zb_lmn
 
+    @optimizable_parameter
+    @property
+    def Rp_lmn(self):
+        """ndarray: Spectral coefficients of R at the cross-section."""
+        return self.xsection.R_lmn
+
+    @Rp_lmn.setter
+    def Rp_lmn(self, Rp_lmn):
+        self.xsection.R_lmn = Rp_lmn
+
+    @optimizable_parameter
+    @property
+    def Zp_lmn(self):
+        """ndarray: Spectral coefficients of Z at the cross-section."""
+        return self.xsection.Z_lmn
+
+    @Zp_lmn.setter
+    def Zp_lmn(self, Zp_lmn):
+        self.xsection.Z_lmn = Zp_lmn
+
+    @optimizable_parameter
+    @property
+    def Lp_lmn(self):
+        """ndarray: Spectral coefficients of Lambda at the cross-section."""
+        return self.xsection.L_lmn
+
+    @Lp_lmn.setter
+    def Lp_lmn(self, Lp_lmn):
+        self.xsection.L_lmn = Lp_lmn
+
+    @property
+    def Lp_basis(self):
```

Comment:
we dont have pass through attributes for the other basis stuff, are these needed? does `eq.xsection.L_basis` not work?